tee-worker/omni-executor/aa-contracts/src/Counter.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+contract Counter {
```

Comment:
Is this part of the project initialization template? should we remove it?

Reply:
We use this contract in tests.